commons-kafka/src/main/scala/com/avsystem/commons/kafka/AbstractSerde.scala

```diff
@@ -0,0 +1,26 @@
+package com.avsystem.commons
+package kafka
+
+import org.apache.kafka.common.serialization.{Deserializer, Serializer}
+
+import java.util
+
+import org.apache.kafka.common.serialization.Serde
```

Comment:
Why the hell didn't they call it `Deser` or `Sedes`?

Reply:
import org.apache.kafka.common.serialization.{Serde => Sedes} ? :P